sql/testdata/subquery

```diff
@@ -61,3 +61,39 @@ query I
 SELECT (SELECT a FROM abc WHERE false)
 ----
 NULL
+
+query I
+VALUES (1, (SELECT (2)))
+----
+1 2
+
+statement ok
+INSERT INTO abc VALUES ((SELECT 7), (SELECT 8), (SELECT 9))
```

Comment:
I think there are two more test cases we need here:
- `INSERT INTO abc VALUES ((SELECT (7,8,9)))`; in postgres, this doesn't type check
- `INSERT INTO abc VALUES ((SELECT 7,8,9))`; in postgres, this errors out with `subquery must return only one column`

Can you add `